server/src/main/java/org/apache/iotdb/db/mpp/execution/FragmentInstanceManager.java

```diff
@@ -148,6 +162,19 @@ public FragmentInstanceInfo abortFragmentInstance(FragmentInstanceId fragmentIns
     return null;
   }
 
+  /** Cancels a FragmentInstance. */
+  public FragmentInstanceInfo cancelTask(FragmentInstanceId instanceId) {
```

Comment:
It seems that `abortFragmentInstance()` do the same thing with `cancelTask`

Reply:
Yes, cancel is called by user kill, abort is called by other FIs' failures in current query.